File: src/main/java/com/post_office/NO5Box.java
package com.post_office;

public class NO5Box extends Box{

    float length = 39.5f;
    float width = 27.5f;
    int height = 23;

    @Override
    boolean validate(float length, float width, int height) {
        if(length < this.length && width < this.width && height < this.height)
            return  true;
        return false;
    }
}

File: src/main/kotlin/post_office/NO5Box.kt
package post_office

class NO5Box(override var length: Float = 39.5f, override var width: Float = 27.5f, override var height: Int = 23) : Box() {
    override fun validate(length: Float, width: Float, height: Int): Boolean {
        return length < this.length && width < this.width && height < this.height
    }
}